src/main/java/io/specto/hoverfly/junit/dsl/RequestTemplateBuilder.java

```diff
@@ -26,13 +26,13 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import static io.specto.hoverfly.junit.core.model.FieldMatcher.fromString;
+import static io.specto.hoverfly.junit.dsl.matchers.ExactMatcher.exactlyMatches;
 import static org.apache.commons.lang3.CharEncoding.UTF_8;
 
 /**
- * A builder for {@link RequestMatcher}
+ * A builder for {@link Request}
  */
-public class RequestMatcherBuilder {
+public class RequestTemplateBuilder {
```

Comment:
Again this is to distinguish it from the actual request matcher. I thought all the requests are now template type?